app/deed/search/routes.py

```diff
@@ -12,7 +11,10 @@ def search_deed():
     def redirect_to_view_deed():
         deed_ref_num = request.form['deedRefNumber']
         url = url_for('deed.view.view_deed', deedRefNum=deed_ref_num)
-        return redirect(url, code=303)
```

Comment:
based on 
http://stackoverflow.com/questions/4764297/difference-between-http-redirect-codes
http://stackoverflow.com/questions/5129076/after-a-post-should-i-do-a-302-or-a-303-redirect
i think 303 is the correct one